config/routes.rb

```diff
@@ -22,6 +22,8 @@
     resources :locations, controller: 'locations'
     resources :offers, controller: 'offers'
 
+    resources :roles, controller: 'roles'
```

Comment:
we only need :index for now, don't we ?
resources :roles, only: :index 
you don't have to specify the controller name ... convention over configuration